src/collections/_documentation/guides/grouping-and-fingerprints/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Grouping & Fingerprints
+sidebar_order: 0
+---
+
+Have you ever had a set of similar-looking issues like this?
+![Issues Dashboard]({% asset guides/grouping-and-fingerprints/issues-dashboard.png @path %})
+
+Let's first understand what makes events group into an issue. testTypeIssue15 is an issue that had 4 events. These events grouped into the issue because they shared the same *fingerprint*
+
+**What is a fingerprint?**
```

Comment:
@MimiDumpling I updated this paragraph to try and make clear Who is setting the Fingerprint. I guess at some point the reader is clued that they can set it themselves, eventually.

Reply:
👍